app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
+	end
+
+	def print_fib(range)
+		range.each do |n|
+			p fib(n)
+		end
+	end
+
+	private
```

Comment:
__[rubocop]__
* Tab detected.
* Indent access modifiers like `private`.
* Keep a blank line before and after `private`.